orchestrator-salt/src/main/resources/salt/salt/sssd/template/add_dns_record.j2

```diff
@@ -22,9 +22,16 @@ for attempt in {1..3}
 do
   sleep 10
   echo "add dns a-record hostname for ${HOSTNAME}, attempt ${attempt}"
+  IPA_STDERR=$(mktemp)
   retVal=0
-  ipa dnsrecord-mod {{ pillar['sssd-ipa']['domain'] }}. "${HOSTNAME}" "--a-rec=${IPADDR}" --ttl {{ pillar['sssd-ipa']['dns_ttl'] }} || retVal=$?
-  if [[ "$retVal" -le 1 ]]; then
+  alreadyExists=0
+  ipa dnsrecord-mod {{ pillar['sssd-ipa']['domain'] }}. "${HOSTNAME}" "--a-rec=${IPADDR}" --ttl {{ pillar['sssd-ipa']['dns_ttl'] }} 2>"${IPA_STDERR}" || retVal=$?
+  cat "${IPA_STDERR}" >&2
+  if grep "ERROR: no modifications to be performed" ${IPA_STDERR} &>/dev/null; then
+    alreadyExists=1
```

Comment:
I think some log would be useful here. Could you please add it?

Reply:
Done